Documentation/compatibility.md

```diff
@@ -44,3 +44,9 @@ The versions of Prometheus compatible to be run with the Prometheus Operator are
 * v2.6.1
 * v2.7.0
 * v2.7.1
+
+## Alertmanager
+
+Any version < v0.15.0 is supported on a best effort basis without end-to-end
```

Comment:
I'd describe it the other way around. Something like:
> We only support Alertmanager v0.16 and above. Everything before that is on a best effort basis.

Reply:
Sounds good. Changed.